wscript_build.py

```diff
@@ -420,10 +420,10 @@ def build(ctx):
         ( "video/out/opengl/cuda_dynamic.c",     "cuda-hwaccel" ),
         ( "video/out/opengl/egl_helpers.c",      "egl-helpers" ),
         ( "video/out/opengl/hwdec_cuda.c",       "cuda-hwaccel" ),
-        ( "video/out/opengl/hwdec_d3d11egl.c",   "d3d-hwaccel" ),
-        ( "video/out/opengl/hwdec_d3d11eglrgb.c","d3d-hwaccel" ),
+        ( "video/out/opengl/hwdec_d3d11egl.c",   "d3d-hwaccel && egl-angle" ),
+        ( "video/out/opengl/hwdec_d3d11eglrgb.c","d3d-hwaccel && egl-angle" ),
         ( "video/out/opengl/hwdec_dxva2gldx.c",  "gl-dxinterop-d3d9" ),
-        ( "video/out/opengl/hwdec_dxva2egl.c",   "d3d9-hwaccel" ),
+        ( "video/out/opengl/hwdec_dxva2egl.c",   "d3d9-hwaccel && egl-angle-win32" ),
```

Comment:
why does this have different dependencies? `egl-angle` works here for me

Reply:
@kevmitch `egl-angle-win32` is originally required by `d3d9-hwaccel` in `wscript` before this commit.

Reply:
@wm4 added this in dd408e68ed57fa5130bef976751e58a5d6e42f75. It no longer appears to be necessary. I tested 

`--disable-angle`
`--disable-d3d9-hwaccel`
`--disable-angle` `--disable-d3d9-hwaccel`

For the sake of simplifying, let's drop the added `egl-angle-win32` dependency. This is front end CLI context creation. Looking at the code, I don't see any clear reason `d3d9-hwaccel` interop alone should require that. 

It would also simplify the logic in ` video/out/gpu/hwdec.c`